src/main/java/esubine/community/auth/model/TokenRepository.java

```diff
@@ -7,7 +7,15 @@
 public interface TokenRepository extends JpaRepository<TokenEntity, Long> {
     TokenEntity findByToken(String token);
 
+//    @Modifying
+//    @Query(value = "delete from token where user_id=:userId", nativeQuery = true)
+//    void deleteAllByUserId(Long userId);
+
     @Modifying
-    @Query(value = "delete from token where user_id=:userId", nativeQuery = true)
-    void deleteAllByUserId(Long userId);
+    @Query("update TokenEntity t set t.isDelete=true " +
+            "WHERE t.userId=:userId ")
+    void setAllByUserId(Long userId);
+
+
+    TokenEntity findByUserId(Long userId);
```

Comment:
가급적 Optional 붙이는걸 권장 합니다.
```suggestion
    Optional<TokenEntity> findByUserId(Long userId);
```